PICMI_Python/fields.py

```diff
@@ -15,6 +15,13 @@ class PICMI_ElectromagneticSolver(_ClassWithInit):
     Electromagnetic field solver
       - grid: grid object to be used by the solver (grid object)
       - method: One of 'Yee', 'CKC', 'Lehe', 'PSTD', 'PSATD', 'GPSTD' or 'ECT' (string)
+        - 'Yee': standard solver using the staggered Yee grid
+        - 'CKC': solver with the extended Cole-Karkkainen stencil with better dispersion properties
```

Comment:
Interestingly, we abbreviate that one with `CK` in openPMD

Reply:
Last C stands for Ben's extension in PRSTAB (2013):
https://doi.org/10.1103/PhysRevSTAB.16.041303